File: cielo24_Java/cielo24/src/cielo24/json/Token.java
package cielo24.json;

import java.util.List;

import static cielo24.Enums.*;

import com.google.gson.annotations.*;

public class Token extends JsonBase {
	@SerializedName("interpolated")
	public boolean interpolated;
	@SerializedName("start_time")
	public int startTime;         // Milliseconds
	@SerializedName("end_time")
	public int endTime;           // Milliseconds
	@SerializedName("value")
	public String value;
	@SerializedName("type")
	public TokenType type;
	@SerializedName("display_as")
	public String typeDisplay;
	@SerializedName("tags")
	public List<Tag> tags;
}

File: cielo24_.NET/Cielo24/JSON/ElementList/Token.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.ElementList
{
    public class Token : JsonBase
    {
        [JsonProperty("interpolated")]
        public bool Interpolated { get; set; }
        [JsonProperty("start_time")]
        public int StartTime { get; set; }      // Milliseconds
        [JsonProperty("end_time")]
        public int EndTime { get; set; }        // Milliseconds
        [JsonProperty("value")]
        public string Value { get; set; }
        [JsonProperty("type")]
        public TokenType Type { get; set; }
        [JsonProperty("display_as")]
        public string TypeDisplay { get; set; }
        [JsonProperty("tags")]
        public List<Tag> Tags { get; set; }
    }
}